messages/teleporter/message_manager.go

```diff
@@ -105,6 +108,22 @@ func (m *messageManager) ShouldSendMessage(warpMessageInfo *vmtypes.WarpMessageI
 	if !ok {
 		return false, fmt.Errorf("relayer not configured to deliver to destination. destinationChainID=%s", destinationChainID.String())
 	}
+
+	// If supportedDestinationss is empty, then all destinations are allowed
```

Comment:
This seems opposite than the expected behavior (one normally assumes that if the allow list is empty, then nothing is allowed).

Reply:
This is similar to how the "allowedRelayerAddresses" field is handled in `TeleporterMessenger`: https://github.com/ava-labs/teleporter/blob/main/contracts/src/Teleporter/TeleporterMessenger.sol#L126

The expected use case here is that if the `supportedDestinations` JSON key is not provided, we don't filter any of the configured destinations. This would be useful if the relayer operator wants messages from subnet A to be relayed to anybody, but messages from subnet B to only be relayed to subnet C, for example.